spec/spec_helper.rb

```diff
@@ -1,8 +1,52 @@
+ENV['RACK_ENV'] = 'test'
+
+require File.join(File.dirname(__FILE__), '..', 'lib/app.rb')
+
+require 'capybara'
+require 'capybara/rspec'
 require 'coveralls'
 require 'simplecov'
+require 'database_cleaner'
+require 'dm-rspec'
+require 'pry'
+require 'rspec'
+require 'tilt/erb'
+
+def app
+  App
+end
+
+Capybara.app = App
+
+RSpec.configure do |config|
+  config.include Capybara::DSL
+  config.include Rack::Test::Methods
```

Comment:
Are you using `rack-test` Can't see it in your `Gemfile` Or am I missing something?